docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = ft.EntitySet('chicago_bike')\n",
+    "\n",
+    "es.entity_from_dataframe(\n",
+    "    dataframe=df.reset_index(),\n",
+    "    entity_id='trips',\n",
+    "    time_index='starttime',\n",
+    "    index='trip_id',\n",
+    ")\n",
+    "                  \n",
+    "es.normalize_entity(\n",
+    "    base_entity_id='trips',\n",
+    "    new_entity_id='from_station_id',\n",
+    "    index='from_station_id',\n",
+    "    make_time_index=False,\n",
+    ")\n",
+    "\n",
+    "es.normalize_entity(\n",
+    "    base_entity_id='trips',\n",
+    "    new_entity_id='weather',\n",
+    "    index='events',\n",
+    "    make_time_index=False,\n",
+    ")                 \n",
+    "\n",
+    "es.normalize_entity(\n",
+    "    base_entity_id='trips',\n",
+    "    new_entity_id='gender',\n",
+    "    index='gender',\n",
+    "    make_time_index=False,\n",
+    ")\n",
+    "\n",
+    "es[\"trips\"][\"gender\"].interesting_values = ['Male', 'Female']\n",
+    "es[\"trips\"][\"events\"].interesting_values = ['tstorms']\n",
+    "es.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calculating the Features\n",
+    "\n",
+    "Now, we can generate features by using a method called Deep Feature Synthesis (DFS). This will automatically build features by stacking and applying mathematical operations called primitives across relationships in an entity set. The more structured an entity set is, the better DFS can leverage the relationships to generate better features. Let’s run DFS using the following parameters:\n",
+    "\n",
+    "* `entity_set` as the entity set we structured previously.\n",
+    "* `target_entity` as stations, since we want to generate features for each station. \n",
+    "* `cutoff_time` as the label times that we generated previously."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, fd = ft.dfs(\n",
+    "    entityset=es,\n",
+    "    target_entity='from_station_id',\n",
+    "    trans_primitives=['hour', 'week', 'is_weekend'],\n",
+    "    cutoff_time=lt,\n",
+    "    cutoff_time_in_index=True,\n",
+    "    include_cutoff_time=False,\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "fm.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two outputs from DFS: a feature matrix and feature definitions. The feature matrix is a table that contains the feature values based on the cutoff times from our labels. Feature definitions are features in a list that can be stored and reused later to calculate the same set of features on future data.\n",
```

Comment:
Might be good to call out that the label values that were created by the labeling process have automatically been appended to the corresponding feature matrix row.